compose-imageviewer/shared/src/androidMain/kotlin/example/imageviewer/view/LocationVisualizer.android.kt

```diff
@@ -6,12 +6,13 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.layout.ContentScale
 import org.jetbrains.compose.resources.ExperimentalResourceApi
 import org.jetbrains.compose.resources.painterResource
+import imageviewer.shared.generated.resources.*
 
 @OptIn(ExperimentalResourceApi::class)
 @Composable
 internal actual fun LocationVisualizer(modifier: Modifier) {
     Image(
-        painter = painterResource("dummy_map.png"),
+        painter = painterResource(Res.drawable.dummy_map),
```

Comment:
There're more such cases:

```
:8080/plus.png:1  
       Failed to load resource: the server responded with a status of 404 (Not Found)
:8080/magic.png:1 
       Failed to load resource: the server responded with a status of 404 (Not Found)
:8080/dummy_map.png:1 
       Failed to load resource: the server responded with a status of 404 (Not Found)
:8080/trash.png:1 
       Failed to load resource: the server responded with a status of 404 (Not Found)
:8080/arrowleft.png:1 
       Failed to load resource: the server responded with a status of 404 (Not Found)
```

I think we can use `painterResource ` for all cases for now - get rid of `painterResourceCached`